content/docs/docs-nav.yaml

```diff
@@ -13,6 +13,8 @@ nav:
     - Inviting users: '/docs/getting-started/user-management/'
     - Adding API specs: '/docs/getting-started/openapi-specs/'
     - Scaffolding components: '/docs/getting-started/scaffolding-components/'
+  - Settings Page:
```

Comment:
The custom plugins heading didn't seem suitable for it. It's also kinda confusing.
 
I thought it would make sense to create a new heading for the settings page where we can add other tutorials in the future. What do you think?

Reply:
The auth proxy is under the settings page.

Reply:
Put it under "Details" for the moment please. We need to overhaul these docs a bit but I'd like to move a few things at once rather than having one doc there on it's own.

Reply:
Ok cool. I'll do that